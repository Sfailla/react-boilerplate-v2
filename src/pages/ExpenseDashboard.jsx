import React, { Fragment } from 'react';
import ExpenseList from '../components/ExpenseList';
import ExpenseListFilters from '../components/ExpenseListFilters';
import ExpensesSummary from '../components/ExpensesSummary';

const ExpenseDashboardPage = () => (
	<Fragment>
		<ExpensesSummary />
		<ExpenseList />
		<ExpenseListFilters />
	</Fragment>
);
export default ExpenseDashboardPage;
