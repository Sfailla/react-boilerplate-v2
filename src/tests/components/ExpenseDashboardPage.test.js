import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboard from '../../pages/ExpenseDashboard';

test('should render ExpenseDashboard', () => {
	const wrapper = shallow(<ExpenseDashboard />);
	expect(wrapper).toMatchSnapshot();
});
