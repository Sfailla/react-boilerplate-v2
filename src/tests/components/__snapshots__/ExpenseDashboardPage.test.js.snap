// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboard 1`] = `
<div>
  <Connect(ExpenseList) />
  <Connect(ExpenseListFilters) />
</div>
`;
